### PR TITLE
PLAT-1252 | create reference icon and color tables

### PR DESCRIPTION
**Pull request by bognix**

### CODE REVIEW

File: database/migrations/2019_05_14_115413_create_references_color_table.php [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+class CreateReferencesColorTable extends Migration
+{
+	/**
+	 * Run the migrations.
+	 *
+	 * @return void
+	 */
+	public function up()
+	{
+		Schema::create('references_color', function (Blueprint $table) {
```
Comment (rogatty) [inline database/migrations/2019_05_14_115413_create_references_color_table.php]: color or colors?

Comment (kvas-damian) [reply database/migrations/2019_05_14_115413_create_references_color_table.php]: `reference_colors`

File: database/migrations/2019_05_14_115422_create_references_icon_table.php [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+class CreateReferencesIconTable extends Migration
+{
+	/**
+	 * Run the migrations.
+	 *
+	 * @return void
+	 */
+	public function up()
+	{
+		Schema::create('references_icon', function (Blueprint $table) {
```
Comment (kvas-damian) [inline database/migrations/2019_05_14_115422_create_references_icon_table.php]: `reference_icons`

File: database/migrations/2019_05_14_115413_create_references_color_table.php (hunk 1 above)
Comment (bognix) [reply database/migrations/2019_05_14_115413_create_references_color_table.php]: it should be refereneces_colors

**Comment by rogatty** [comment]
What about icons seeder?

**Comment by bognix** [comment]
@rogatty @kvas-damian will add those because he has better knowledge of what should be there as he is the one adding the icons right now.

**Pull request merged**